code/src/components/MovieDetails.jsx

```diff
@@ -0,0 +1,128 @@
+import React, { useState, useEffect } from "react";
+import { useParams, Link  } from "react-router-dom";
+import { API_KEY, MOVIE_DETAILS_URL } from "../utils/urls";
+import styled from 'styled-components'
+import { FiChevronLeft } from "react-icons/fi";
+
+const DetailsContainer = styled.div`
+  min-height: 100vh;
+  background: center center;
+  background-size: cover;
+  position: relative;
+  display: flex;
+  align-items: flex-end;
+  padding: 1rem;
+`;
+
+const Button = styled.button`
+  position: absolute;
+  display: inline-flex;
+  color: #fff;
+  background: #000;
+  font-weight: 700;
+  text-decoration: none;
+  border: 1px solid #fff;
+  padding: 5px 20px;
+  left: 17px;
+  top: 17px;
+  text-transform: uppercase;
+  font-size: 16px;
+  cursor: pointer;
+  text-align: center;
+  svg {
+    position: relative;
+    top: 50%;
+    left: 0%;
+    transform: translate(-50%, 25%);
+  }
+  @media (min-width: 767px) {
+    font-size: 18px;
+  }
+  @media (min-width: 1024px) {
+    font-size: 20px;
+  }
+`;
+
+const MovieDetailsSummary = styled.div`
+  display: flex;
+  flex-direction: column;
+  margin-bottom: 1rem;
+  width: 90%;
+  img {
+    width: 200px;
+    height: auto;
+    border-radius: 10px;
+  }
+  @media (min-width: 767px) {
+    flex-direction: row;
+    img{
+      height: 50vh;
+      width: auto;
+    }
+  }
+  @media (min-width: 1024px) {
+    flex-direction: row;
+    img{
+      height: 50vh;
+      width: auto;
+    }
+  }
```

Comment:
It's not necessary to have the media query for min 1024px as they are the same as for min 767px, but it is easier to read and see differences and similarities between different sizes maybe?